sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessage.java

```diff
@@ -353,7 +392,10 @@ public OffsetDateTime getScheduledEnqueueTime() {
      * @see #getScheduledEnqueueTime()
      */
     public ServiceBusMessage setScheduledEnqueueTime(OffsetDateTime scheduledEnqueueTime) {
-        this.scheduledEnqueueTime = scheduledEnqueueTime;
+        if (scheduledEnqueueTime != null) {
```

Comment:
What if they're trying to clear the scheduledEnqueueTime? Isn't null a value option to pass? Same with other instances.

Reply:
Internally these key,value pair Map and  Null value is not allowed. 
The code on master is also checking for `null`  and not putting in map is user provided null value.
User can get `Map` using `amqpAnnotatedMessage.getMessageAnnotations()` and remove a key if they do not want it.

Reply:
May be worthwhile to document here. it's odd having one method do one thing and then the other do something else.